<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternWindow</class>
 <widget class="QWidget" name="PatternWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>603</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Record a pattern</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="newPatternLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Record a new Pattern</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="patternNameLayout">
       <item>
        <widget class="QLabel" name="patternNameLabel">
         <property name="text">
          <string>Pattern Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="patternNameField"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="recordingsLayout"/>
     </item>
     <item>
      <widget class="IODevicePlotterWidget" name="ioPlotter"/>
     </item>
     <item>
      <widget class="QLabel" name="recordingsLabel">
       <property name="text">
        <string>Add at least 5 recordings of you saying the phrase</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="recordingsListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="recordingLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="toggleRecordingButton">
         <property name="text">
          <string>Start Recording</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRecordingButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playRecordingButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Play Recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionLayout">
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IODevicePlotterWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">iodeviceplotterwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>patternNameField</tabstop>
  <tabstop>toggleRecordingButton</tabstop>
  <tabstop>playRecordingButton</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>PatternWindow</sender>
   <signal>createButtonEnabledChanged(bool)</signal>
   <receiver>createButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PatternWindow</sender>
   <signal>deleteButtonEnabledChanged(bool)</signal>
   <receiver>deleteRecordingButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PatternWindow</sender>
   <signal>playButtonEnabledChanged(bool)</signal>
   <receiver>playRecordingButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PatternWindow</sender>
   <signal>toggleButtonEnabledChanged(bool)</signal>
   <receiver>toggleRecordingButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PatternWindow</sender>
   <signal>playButtonEnabledChanged(bool)</signal>
   <receiver>recordingsListWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PatternWindow</sender>
   <signal>cancelButtonEnabledChanged(bool)</signal>
   <receiver>cancelButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>playButtonEnabledChanged(bool)</signal>
  <signal>deleteButtonEnabledChanged(bool)</signal>
  <signal>toggleButtonEnabledChanged(bool)</signal>
  <signal>createButtonEnabledChanged(bool)</signal>
  <signal>cancelButtonEnabledChanged(bool)</signal>
 </slots>
</ui>
